Makefile

```
@@ -66,14 +67,24 @@ misspell:	$(MISSPELL)
 misspell-correction:	$(MISSPELL)
 	$(MISSPELL) -w $(ALL_DOCS)
 
+.PHONY: normalized-link-check
+normalized-link-check:
+	@if grep -R '\.\./docs/' docs model/database; then \
+		echo "\nERROR: Found occurrences of '../docs/'; see above."; \
+		echo "       Remove the '../docs/' from doc page links referencing doc pages."; \
+		exit 1; \
+	else \
+		echo "Normalized-link check passed."; \
+	fi
+
 .PHONY: markdown-link-check
-markdown-link-check:
+markdown-link-check: normalized-link-check
 	docker run --rm \
 		--mount 'type=bind,source=$(PWD),target=/home/repo' \
 		lycheeverse/lychee \
 		--config home/repo/.lychee.toml \
 		--root-dir /home/repo \
-		--verbose \
+		--verbose $(MARKDOWN_LINK_CHECK_ARG) \
```

nit:

```suggestion
		--verbose \
		$(MARKDOWN_LINK_CHECK_ARG) \
```

just to make it more obvious it's not the value  of `--verbose`